.env.example

```
@@ -69,11 +69,12 @@ API_PORT=3010
 API_HOST=0.0.0.0
 
 # CORS configuration
-CORS_ORIGIN=http://localhost:3011
+CORS_ORIGIN=http://localhost:3100
 CORS_CREDENTIALS=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CORS_CREDENTIALS key should go before the CORS_ORIGIN key

(UnorderedKey)